agama-lib/src/proxies.rs

```diff
@@ -121,9 +117,9 @@ trait Language1 {
 }
 
 #[dbus_proxy(
-    interface = "org.opensuse.DInstaller.Questions1",
-    default_service = "org.opensuse.DInstaller.Questions",
-    default_path = "/org/opensuse/DInstaller/Questions1"
+    interface = "org.opensuse.Agama.Questions1",
+    default_service = "org.opensuse.Agama.Questions1",
```

Comment:
`Questions` iface is not renamed yet? It should be `Agama1.Questions` like `Agama1.Validation`.

Reply:
You are right! Questions has its own service.